styles.css

```diff
@@ -136,14 +478,195 @@ ul.navbar-items {
   }
 
   .prof {
-    color: white;
-    font-size: 25px;
+    font-family: 'Inter', sans-serif;
+    color: #fff5e1;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
   }
 
   .desc {
     color: #f4f5f7;
     width: 300px;
-    line-height: 1.5rem;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px !important;
+  }
+
+  /* works-section responsive style  */
+
+  #works-section {
+    margin-bottom: 100px;
+    overflow-x: hidden;
+  }
+
+  .box-container {
+    flex-direction: column;
+    margin: 5px 10px;
+  }
+
+  .left-section {
+    margin-bottom: 10px;
+  }
+
+  .left-section img {
+    float: none;
+    width: 100% !important;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    gap: 10px;
+    margin: 0;
+    padding-left: 4%;
+    padding-right: 4%;
+  }
+
+  .first-desc {
+    text-align: left;
+    margin: 10px;
+  }
+
+  .first-desc h3 {
+    font-size: 16px;
+  }
+
+  .first-desc ul {
+    padding-left: 0;
+  }
+
+  .first-desc p {
+    font-size: 14px;
+  }
+
+  .first-desc ul li {
+    display: inline-block;
+    margin: 10px 0;
+    padding-left: 10px;
+  }
+
+  .first-desc button {
+    margin-top: 20px;
+    color: #f4f5f7;
+  }
+
+  .hr {
+    margin: 20px;
+    font-family: 'Crete Round', sans-serif;
+    color: #1c1a19;
+  }
+
+  hr {
+    position: relative;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: #212121;
+  }
+
+  .item-child button {
+    display: block;
+    background-color: #f55800;
+    width: 100%;
+    height: 50px;
+    margin-bottom: -30px;
+  }
+
+  .grid-btn {
+    border: none;
+    font-family: 'Crete Round', sans-serif;
+    font-size: 17px;
+    color: #f4f5f7;
+    text-align: center;
+  }
+
+  /* about-section responsive styles */
+
+  #about-page-container {
+    background-color: #1c1a19;
+    padding: 20px;
+  }
+
+  .about-me-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 10px;
+  }
+
+  .about-me-info {
+    margin: 50px 0 0 0;
+    padding: 0 10px;
+  }
+
+  .about-me-info h2 {
+    font-size: 30px;
+    margin-bottom: 10px;
+    color: #f55800;
+  }
+
+  .about-me-info p {
+    color: #f4f5f7;
+    font-size: 18px;
+  }
+
+  .about-me-info button {
+    color: white;
+    padding: 10px;
+    background-color: #f55800;
+    border: none;
+  }
+
+  .about-me-image {
+    flex: 1;
+    width: 100%;
+    height: auto;
+    margin-right: 250px !important;
```

Comment:
- **[OPTIONAL]** KIndly avoid to use **!important** in your styles for the best practices of HTML & CSS.